app/services/stats/r401_offender_sar_closed_cases_report.rb

```diff
@@ -15,13 +15,13 @@ def etl_handler
     end
 
     def case_scope
-      CaseFinderService.new(@user).closed_cases_scope.where(type: 'Case::SAR::Offender')
+      CaseFinderService.new(@user).closed_cases_scope.where("type LIKE 'Case::SAR::Offender%'")
```

Comment:
if a new report for complaint case report is introduced below,  I would think this one shouldn't be changed